package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "react-native-image-keyboard",
   "title": "React Native Image Keyboard",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

```suggestion
  "version": "1.1.0",
```

Please revert as versioning is not done on PRs.